deploy-options.sh

```diff
@@ -40,37 +40,21 @@ function deploy_options {
     echo "| TryOutApps |"
     echo "--------------"
     echo
-    echo "[0] All"
-    echo "[1] Staging"
-    echo "[2] UAT"
-    echo "[3] Production"
+    echo "[0] Staging"
+    echo "[1] UAT"
+    echo "[2] Production"
     echo
     echo "=================="
     echo
     echo "---------------------"
     echo "| APP STORE CONNECT |"
     echo "---------------------"
     echo
-    echo "[4] App Store"
+    echo "[3] App Store"
```

Comment:
Can we make this dynamic? We define possible envs somewhere at the start of the script, for example as dictionary where key is environment's id (e.g. `"internal-staging"` and value is the display name, e.g. `"Staging"`). This way we don't have to fix prints every time we fiddle with the array of options.

Reply:
I would like to keep this one as-is. The user should define this part of the script so they can write whatever text/format they want here. 

And this is only an example/template that they have to change.